src/openforms/submissions/models.py

```diff
@@ -253,6 +255,30 @@ def get_merged_data(self) -> dict:
 
         return merged_data
 
+    def get_printable_data(self) -> Dict[str, str]:
```

Comment:
Fine for now, but if we keep expanding on this, we might have to just register a display/print function which takes the `info` argument and have a mapping of `type: print_function` so that can be tested/maintained in a bit more isolated way.